data-raw/jpnprefs.R

```diff
@@ -92,10 +96,54 @@ jpnprefs %<>%
   select(jis_code, prefecture, capital, region, major_island, capital_latitude = latitude, capital_longitude = longitude) %>%
   as_tibble()
 
+# ---- English region and island names
+url <- "https://en.wikipedia.org/wiki/Prefectures_of_Japan"
+
+session <- bow(url)
+
+jpn_pref_raw <- scrape(session) %>%
+  html_nodes("table.wikitable:nth-child(49)") %>%
+  #.[[1]] %>%
+  html_table() %>%
+  purrr::flatten_df()
+
+jpn_pref_df <- jpn_pref_raw %>%
+  janitor::clean_names() %>%
```

Comment:
I do not feel motivated to use janitor for this process. 
Is it possible to change to a method using `dplyr::select()` which explicitly selects and renames a variable?